### PR TITLE
Fix eslint errors

### PR DESCRIPTION
**Pull request by vovikdrg**
I see that my previous PR break a build.

I tried to fix as max as i can but i have disabled some eslint rules for specific lines

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ osx_image: xcode8
 
 install:
     - npm install
-    - npm install appium@1.5.3
+    - npm install appium@1.7.2
```
Comment (rhysd) [inline .travis.yml:10]: I guess this does not work because this repo is not supporting the version yet.

File: lib/launcher.js [hunk 2]
```diff
@@ -8,16 +8,16 @@ import fs from 'fs';
...
-type AppiumArgs = {
-    [name: string]: string | boolean | null;
+type AppiumArgsType = {
+    [name: string]: string | boolean | null
 };
 
-type AppiumServiceConfig = {
-    appium?: {
+type AppiumServiceConfigType = {
+    appium?: { // eslint-disable-line flowtype/delimiter-dangle
```
Comment (rhysd) [inline lib/launcher.js]: Why is the rule disabled?

File: lib/launcher.js [hunk 3]
```diff
@@ -58,8 +58,8 @@ class AppiumLauncher {
         }
     }
 
-    _startAppium (): Promise<child_process$ChildProcess> {
-        return new Promise((resolve, reject) => {
+    _startAppium (): Promise<child_process$ChildProcess> { // eslint-disable-line camelcase
+        return new Promise((resolve: any, reject: any) => {
```
Comment (rhysd) [inline lib/launcher.js]: `resolve` and `reject` should not be typed as `any`. Their types are function which takes one argument and returns nothing.

File: lib/launcher.js [hunk 4]
```diff
@@ -92,7 +92,7 @@ class AppiumLauncher {
         });
     }
 
-    _getAppiumFileName(){
+    _getAppiumFileName (): string {
```
Comment (rhysd) [inline lib/launcher.js:95]: Doesn't Flow infer the return type from the function's return expression?

**Review by rhysd — CHANGES_REQUESTED** [review]

File: lib/launcher.js [hunk 5]
```diff
@@ -160,7 +160,7 @@ class AppiumLauncher {
         }
         return ret;
     }
-    
+ // eslint-disable-line    no-trailing-spaces
```
Comment (rhysd) [inline lib/launcher.js]: Why is this rule disabled?

**Review by rhysd — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (vovikdrg) [reply .travis.yml:10]: Is it ok if I will make it work with latest?

File: lib/launcher.js (hunk 4 above)
Comment (vovikdrg) [reply lib/launcher.js:95]: Yes there was exception that function does not have return type

File: lib/launcher.js (hunk 5 above)
Comment (vovikdrg) [reply lib/launcher.js]: I was not able to fix this, and its probably due to windows. I am working on mac now to solve other problems

File: lib/launcher.js (hunk 2 above)
Comment (vovikdrg) [reply lib/launcher.js]: Fixed

File: lib/launcher.js (hunk 3 above)
Comment (vovikdrg) [reply lib/launcher.js]: Fixed

File: lib/launcher.js (hunk 5 above)
Comment (vovikdrg) [reply lib/launcher.js]: Fixed

File: .travis.yml (hunk 1 above)
Comment (rhysd) [reply .travis.yml:10]: Yes, it's so helpful to make it work.

File: lib/launcher.js (hunk 4 above)
Comment (rhysd) [reply lib/launcher.js:95]: I got it.

**Review by rhysd — APPROVED** [review]

**Comment by rhysd** [comment]
From the CI result, `npm run lint` passed successfully. So I'm merging this PR. Although tests are not passing, it is not a scope of this PR.

**Comment by rhysd** [comment]
@vovikdrg Thank you for your patch. I merged.

**Comment by vovikdrg** [comment]
@rhysd  do you have any ideas why Travis could fail? I've tested on windows and Mac it works on both. I want to make build green :)

**Comment by rhysd** [comment]
@vovikdrg 

https://travis-ci.org/rhysd/wdio-appium-service/builds/331665843#L322

Error message is

> ERROR: connect ECONNREFUSED 127.0.0.1:4723

I'm not sure but ECONNREFUSED occurs generally by

- the server is already started (port is in use)
- client is connecting the server but server was not starte

**Pull request merged**